Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -869,6 +869,7 @@ class BrowserViewController: UIViewController {
             guard let loading = change?[NSKeyValueChangeNewKey] as? Bool else { break }
             toolbar?.updateReloadStatus(loading)
             urlBar.updateReloadStatus(loading)
+            if (!loading) {runScriptsOnWebView(webView)}
```

Comment:
Nit: to follow our formatting conventions, please put the `runScriptsOnWebView` on a new line (like we do for the `log.debug` right below here).